src/views/components/home/square-photos.twig

```diff
@@ -22,7 +22,9 @@
     <div class="grid {{ items|length <= 3 ? 'one-row md:grid-cols-' ~ items|length : 'md:grid-cols-3 two-row' }} grid-flow-row gap-3 sm:gap-8">
         {% for item in items %}
             <a class="banner-entry square-photos {{ item.text ? 'has-overlay with-hover' : '' }}" href="{{ item.url }}" {% if item.link_type == 'external_link' %}target="_blank"{% endif %}>
-                <div class="lazy__bg lazy" data-bg="{{ item.image.url }}"></div>
+                <div class="lazy__bg lazy bg-no-repeat" data-bg="{{ item.image.url }}"
+                style="background-size: {{ theme.settings.get('squar_photo_bg_image_size') }};"
```

Comment:
Give it default value
`theme.settings.get('squar_photo_bg_image_size', 'contain')`